transcribe-results.php

```diff
@@ -5,7 +5,7 @@
  * @package wp-softcatala
  */
 wp_enqueue_script( 'sc-js-transcribe-results', get_template_directory_uri() . '/static/js/transcribe-results.js', array('sc-js-main'), WP_SOFTCATALA_VERSION, true );
-wp_enqueue_script( 'sc-js-otranscribe', get_template_directory_uri() . '/static/js/otranscribe.js', array('sc-js-main'), WP_SOFTCATALA_VERSION, true );
+wp_enqueue_script( 'sc-js-otranscribe', get_template_directory_uri() . '/static/js/otranscribe.c73d4173820839f2b8f6.js', array('sc-js-main'), WP_SOFTCATALA_VERSION, true );
```

Comment:
The hash is not needed. Whenever we deploy a new version of the code to production, a new query string parameter with the value of `WP_SOFTCATALA_VERSION` changes, replacing the need of any hash